components/prismic/SliceLoader.vue

```diff
@@ -3,7 +3,8 @@
     <div v-for="slice in slices">
         <color-quote v-if="slice.slice_type === 'quote_with_color'" :fields="slice.primary"></color-quote>
         <split-callout  v-else-if="slice.slice_type === 'split_callout'" :fields="slice.primary"></split-callout>
-      <call-to-action v-else-if="slice.slice_type === 'call_to_action'" :fields="slice.primary"></call-to-action>
+        <call-to-action v-else-if="slice.slice_type === 'call_to_action'" :fields="slice.primary"></call-to-action>
+        <text-block v-else-if="slice.slice_type === 'text'" :fields="slice.primary"></text-block>
```

Comment:
The TextBlock component needs to be added to the SliceLoader for any component to be rendered on the page.  The key here is the SliceType, in this case `text`